### PR TITLE
update gradle files

### PR DESCRIPTION
**Pull request by hlwhl**

### CODE REVIEW

File: fingerprint-dialog-compat/build.gradle [hunk 1]
```diff
@@ -10,8 +10,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'com.github.dcendents.android-maven'
 
 android {
-    compileSdkVersion 'android-P'
-    buildToolsVersion '28.0.0 rc1'
+    compileSdkVersion 28
```
Comment (kevalpatel2106) [inline fingerprint-dialog-compat/build.gradle]: We should add `buildToolsVersion '28.0.0'` here.

File: app/build.gradle [hunk 2]
```diff
@@ -16,8 +16,7 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 'android-P'
-    buildToolsVersion '28.0.0 rc1'
+    compileSdkVersion 28
```
Comment (kevalpatel2106) [inline app/build.gradle]: We should add `buildToolsVersion '28.0.0'` here.

**Review by kevalpatel2106 — CHANGES_REQUESTED** [review]

**Comment by kevalpatel2106** [comment]
@hlwhl Can you check why the CI is failing?

**Review by hlwhl — COMMENTED** [review]
added

File: app/build.gradle (hunk 2 above)
Comment (hlwhl) [reply app/build.gradle]: added

File: fingerprint-dialog-compat/build.gradle (hunk 1 above)
Comment (hlwhl) [reply fingerprint-dialog-compat/build.gradle]: added

**Review by kevalpatel2106 — APPROVED** [review]

**Comment by kevalpatel2106** [comment]
@hlwhl This PR will be added in the next release. Thanks.

**Pull request merged**